src/python/examples/CMakeLists.txt

```diff
@@ -56,6 +56,9 @@ if(${TRITON_ENABLE_PYTHON_GRPC})
       grpc_explicit_int_content_client.py
       grpc_explicit_int8_content_client.py
       grpc_image_client.py
+      simple_grpc_aio_infer_client.py
+      simple_grpc_aio_sequence_stream_infer_client.py
+      simple_grpc_aio_test_client.py
```

Comment:
Instead of `simple_grpc_aio_test_client.py` in example, can you add these tests as python unittest cases? 
For example: https://github.com/triton-inference-server/server/blob/main/qa/L0_cuda_graph/trt_cuda_graph_test.py

Reply:
Moved